language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/type_test_expression_ctx_config8.json

```diff
@@ -444,15 +444,15 @@
     {
       "label": "module1:TestMap2",
       "kind": "TypeParameter",
-      "detail": "Map",
+      "detail": "Typereference",
```

Comment:
@malinthar IMO, the detail here should be `Map`. Do you have an idea why this changed here?